.github/CODEOWNERS

```diff
@@ -233,6 +233,7 @@
 /core/templates/components/common-layout-directives/common-elements/background-banner.component.ts @srijanreddy98
 /core/templates/components/common-layout-directives/common-elements/background-banner.component.html @srijanreddy98
 /core/templates/components/common-layout-directives/common-elements/confirm-or-cancel-modal.controller*.ts @srijanreddy98
```

Comment:
Do we need to remove this line?

@sajalasati, Should we add a linter check to ensure that all the lines in the CODOEWNERS correspond to at least one file?

Reply:
These files have not been deleted.

Reply:
The plan is to migrate all modal instances separately. I have made an angular version of the base class which can then be used in the other modals. This file will be deleted once all the modals are migrated. Also, the lint check reports if the pattern doesn't match a file :)

Reply:
> Do we need to remove this line?
> 
> @sajalasati, Should we add a linter check to ensure that all the lines in the CODOEWNERS correspond to at least one file?

@DubeySandeep We already have that check in `codeowner_linter` file (see [here](https://github.com/oppia/oppia/blob/develop/scripts/linters/codeowner_linter.py#L264-L283)) to raise error for files that do not correspond to a file/dir. Reading the follow-up comments now, I think the linter is working as expected.